docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1568 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !lscpu
</pre><p>FYI: This benchmarks a CPU released in 2019 against a GPU made available in 2022-2023. I saw that the website benchmarks Allan collected also used an AMD EPYC 7642, which is also a 2019-2020 era CPU. Direct comparisons are always tough to make in a fair way, but if we have access to a newer server CPU, that'd be ideal.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>